modules/users/manifests/dennistsiang.pp

```diff
@@ -0,0 +1,8 @@
+# Create the dennistsiang user
+class users::dennistsiang {
+  govuk_user { ‘dennistsiang’:
```

Comment:
Looks like we have "fancy" quotes in here and the puppet parser doesn't like them.  Canyou replace with "normal" single quotes.